experiments/AMIP/components/atmosphere/climaatmos.jl

```diff
@@ -110,17 +75,78 @@ function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
     return sim
 end
 
+"""
+    get_model_prog_state(sim::ClimaAtmosSimulation)
+
+Extension of Checkpointer.get_model_prog_state to get the model state.
+"""
+function get_model_prog_state(sim::ClimaAtmosSimulation)
+    return sim.integrator.u
+end
+
+"""
+    get_field(atmos_sim::ClimaAtmosSimulation, ::Val{:radiative_energy_flux_toa})
+
+Extension of Interfacer.get_field to get the net TOA radiation, which is a sum of the
+upward and downward longwave and shortwave radiation.
+"""
```

Comment:
I noticed recently that sometimes the docstrings of extensions can sometimes cause the extensions to be ignored. It doesn't seem to be problematic here, but we should probably decide on a convention (Clima wide) as to how to deal with this. (not as part of this PR)

Reply:
good point!